third_party/terraform/website/docs/d/datasource_google_secret_manager_secret_version.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_secret_manager_secret_version"
+sidebar_current: "docs-google-datasource-secret-manager-secret-version"
+description: |-
+  Get a Secret Manager secret's version.
+---
+
+# google\_secret\_manager\_secret\_version
+
+Get a Secret Manager secret's version. For more information see the [official documentation](https://cloud.google.com/secret-manager/docs/) and [API](https://cloud.google.com/secret-manager/docs/reference/rest/v1beta1/projects.secrets.versions).
+
+## Example Usage
+
+```hcl
+data "google_secret_manager_secret_version" "basic" {
+  provider = google-beta
+  secret = "my-secret"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Optional) The project to get the secret version for. If it
+    is not provided, the provider project is used.
+
+* `secret` - (Required) The secret to get the secret version for.
+
+* `version` - (Optional) The version of the secret to get. If it
+    is not provided, the latest version is retrieved.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `secret_data` - The secret data. No larger than 64KiB. A base64-encoded string.
+
+* `name` - The resource name of the SecretVersion. Format:
+  `projects/{{project}}/secrets/{{secret_id}}/versions/{{version}}`
+
+* `create_time` - The time at which the Secret was created.
+
+* `destroy_time` - The time at which the Secret was destroyed. Only present if state is DESTROYED.
+
+* `state` - The current state of the SecretVersion.
```

Comment:
Should I change this to be `enabled` to be consistent with the resource?  Or since it's read-only, does it matter?

Reply:
I'd be consistent with the resource, but I don't think it's a big deal?